### PR TITLE
updates to work with windows

### PR DESCRIPTION
**Pull request by AllisonBolen**
# Summary: 
- update readme to include dbeaver mention 
- update envconfig.go to work on windows

# Justification: 
- we paired on setup and you had me use dbeaver to access the database
- the envconfig.go file had issues running the MkDir command so I updated it to MkDirAll

# Docs and Resources: 
- NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,26 +14,39 @@ This is a work in progress...
...
-2. initialize the project from the root directory
+
+#### *Connect to MySql using DBeaver: 
+   1. install Dbeaver if you haven't already
+   2. connect to the MySql Instance: 
+     1. HOST: `localhost:3308`
+     2. DB: `minuchin`
+     3. Username: `root`
+     4. Password: _obtain from docker yaml file_
+   3. *there was a step here that I don't remember, something to do with drivers or ports????*
```
Comment (mftcodes) [inline README.md:40]: Under connection settings > driver properties > set `allowPublicKeyRetrieval` to `TRUE`

File: config/envConfig.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 
 func SetupLocalDevEnv() error {
 	tempDir := SetLocalConfDir()
-	err := os.Mkdir(tempDir, 0750)
+	err := os.MkdirAll(tempDir, 0750)
```
Comment (mftcodes) [inline config/envConfig.go:10]: This is a great add and a clear miss on my part

File: README.md [hunk 3]
```diff
@@ -14,26 +14,39 @@ This is a work in progress...
...
 
-NOTE: You must have the API running in order for the front end to function.
+For Windows OS we also recommend installing [Docker Desktop](https://www.docker.com/products/docker-desktop/) and [DBeaver](https://dbeaver.io/download/)
 
-### Go Back End API, Option 1
+NOTE: You must have the API running in order for the front end to function.
 
-1. setup MySQL in Docker
+### Database Setup: 
+1. Setup MySQL in Docker
```
Comment (mftcodes) [inline README.md:22]: Rather than an ordered list item, maybe we should do a sub-header here?

**Review by mftcodes — CHANGES_REQUESTED** [review]
A couple comments to look at, otherwise this is great!

File: README.md (hunk 3 above)
Comment (AllisonBolen) [reply README.md]: add as sub heading

**Review by mftcodes — APPROVED** [review]

**Pull request merged**